### PR TITLE
Publish Attempted QPS from Driver

### PR DESCRIPTION
**Pull request by ncteisen**
Needed for an internal benchmark dashboarding change

### CODE REVIEW

**Comment by grpc-testing** [comment]
```
****************************************************************

libgrpc.so

     VM SIZE        FILE SIZE
 ++++++++++++++  ++++++++++++++

  [ = ]       0        0  [ = ]


****************************************************************

libgrpc++.so

     VM SIZE        FILE SIZE
 ++++++++++++++  ++++++++++++++

  [ = ]       0        0  [ = ]



```

**Comment by grpc-testing** [comment]
```
[trickle] No significant performance differences
```

**Comment by grpc-testing** [comment]
```
[microbenchmarks] No significant performance differences
```

File: src/proto/grpc/testing/control.proto [hunk 1]
```diff
@@ -214,6 +214,8 @@ message Scenarios {
 // once the scenario has finished.
 message ScenarioResultSummary
 {
+  // Requested QPS for fixed load tests.
+  double attempted_qps = 19;
```
Comment (vjpai) [inline src/proto/grpc/testing/control.proto]: I think we seem to sort proto by tag, not alphabetically, wdyt?

File: test/cpp/qps/qps_json_driver.cc [hunk 2]
```diff
@@ -86,6 +86,7 @@ static std::unique_ptr<ScenarioResult> RunAndReport(const Scenario& scenario,
   result->mutable_scenario()->CopyFrom(scenario);
 
   GetReporter()->ReportQPS(*result);
+  GetReporter()->ReportAttemptedQPS(*result);
```
Comment (vjpai) [inline test/cpp/qps/qps_json_driver.cc]: This is ~~chicken nuggets~~ a constant, right? Why report something that's just part of the client config which is the input to the scenario?

File: src/proto/grpc/testing/control.proto (hunk 1 above)
Comment (ncteisen) [reply src/proto/grpc/testing/control.proto]: Done

File: test/cpp/qps/qps_json_driver.cc (hunk 2 above)
Comment (ncteisen) [reply test/cpp/qps/qps_json_driver.cc]: It doesn't need to be logged I guess, but it does need to exist in the ReportSummary proto. This is to match internal behavior for the Stubby benchmarking system.

Both sent the report protos to a Result Server, which is responsible for writing the data somewhere where perfkit can read it (for internal dashboards).

Ultimate goal is to be able to dashboard open loop scenarios

**Comment by ncteisen** [comment]
Addressed comments, will ping you internal bug info off github

**Comment by grpc-testing** [comment]
```
****************************************************************

libgrpc.so

     VM SIZE        FILE SIZE
 ++++++++++++++  ++++++++++++++

  [ = ]       0        0  [ = ]


****************************************************************

libgrpc++.so

     VM SIZE        FILE SIZE
 ++++++++++++++  ++++++++++++++

  [ = ]       0        0  [ = ]



```

**Comment by grpc-testing** [comment]
```
[trickle] No significant performance differences
```

**Comment by vjpai** [comment]
Ok, so in this case attempted_qps is something similar to time that is also being reported even though it's more or less input (though not exactly since it could be variable depending on when the marks get processed). Longer term, maybe consider putting the entire scenario config into the result summary? It's in the ScenarioResult, so it seems like it would make sense for it to be in the ScenarioResultSummary also. (kermit_sips_tea)

**Comment by grpc-testing** [comment]
```
[microbenchmarks] No significant performance differences
```

**Comment by ncteisen** [comment]
I have accomplished this in a better way. Closing.

(shame to lose a purely green PR)

**Pull request closed without merging**